cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
where do you use this file? can't find it

Reply:
i think was auto generated when i ran cypress. fixing it in next commit